src/StatusQ/Components/StatusChatList.qml

```diff
@@ -96,31 +94,27 @@ Column {
                 StatusChatListItem {
 
                     id: statusChatListItem
-
                     property string profileImage: ""
-
                     opacity: dragSensor.active ? 0.0 : 1.0
                     Component.onCompleted: {
                         if (typeof statusChatList.profileImageFn === "function") {
                             profileImage = statusChatList.profileImageFn(model.chatId || model.id) || ""
                         }
                     }
                     originalOrder: model.position
-                    chatId: model.chatId || model.id
-                    categoryId: model.categoryId || ""
+                    chatId: model.itemId || model.id
+                    categoryId: model.itemId || ""
                     name: !!statusChatList.chatNameFn ? statusChatList.chatNameFn(model) : model.name
-                    type: model.chatType
+                    type: !!model.type ? model.type : StatusChatListItem.Type.CommunityChat
                     muted: !!model.muted
-                    hasUnreadMessages: !!model.hasUnreadMessages || model.unviewedMessagesCount > 0
-                    hasMention: model.mentionsCount > 0
-                    badge.value: model.chatType === StatusChatListItem.Type.OneToOneChat ?
-                        model.unviewedMessagesCount || 0 :
-                        model.mentionsCount || 0
-                    selected: (model.chatId || model.id) === statusChatList.selectedChatId
+                    hasUnreadMessages: !!model.hasNotification || model.unviewedMessagesCount > 0
+                    hasMention: model.notificationsCount > 0
+                    badge.value: model.notificationsCount
```

Comment:
how do we know whether hasUnreadMessages or hasMention?

Reply:
@alexandraB99 as display says, 
- if `model.hasNotification` is **false** then **don't display** neither `dot badge` nor `count badge` **regardless to the `model. notificationsCount`** (but `model.notificationsCount` in such cases will be surely 0)
- if `model.hasNotification` is **true and `model.notificationsCount` is 0**, that means we have some notification but not direct to the logged in user (so far that means user has some unread messages in that channel) so you should **display `dot badge`**
- if `model.hasNotification` is **true and `model.notificationsCount` is greater 0** then **display `count badge`**

so as you may see `model.hasNotification` informs qml that we should display either dot or count badge, but `model.countNotification` determines if it will be dot badge (`model.countNotification` == 0) or count badge (`model.countNotification` > 0), *respecting the advantage that count badge has a higher priority*.

Reply:
Almost correct (two fixes):

1. When `hasNotification && notificationCount == 0` (case: has unread messages) we **don't** render a dot badge. The badge is **only** shown when `notifiactionCount > 0`. Maybe you confuse it with the badge rendered on the navigation tab bar icons, but this is individual chat items.
2.  When `hasNotification && notificationCount > 0` (case: unread mentions) then the chat item is **bold** as well.

So all in all there's just 2 cases (ignoring muted, inactive, normal and selected chat list items:

1. Text is bold (unread messages)
2. Text is bold + badge (mentions)

Reply:
> Text is bold (unread messages)
Text is bold + badge (mentions)

Exactly. :)